libc/src/__support/RPC/rpc.h

```diff
@@ -30,6 +28,17 @@
 namespace LIBC_NAMESPACE_DECL {
 namespace rpc {
 
+/// Use scoped atomic variants if they are available for the target.
+#if !__has_builtin(__scoped_atomic_load_n)
+#define __scoped_atomic_load_n(src, ord, scp) __atomic_load_n(src, ord)
```

Comment:
you might want to guard all the arguments with `()`?

Reply:
These get undefed right after so I'm not too concerned about misuse.

Reply:
Even for future-proofing, if someone else contributes to this and doesn’t realize it’s a macro or doesn’t use it correctly, I still believe it’s a good idea to implement it in a proper way. Not sure about the policy of libc project though. I'll leave it to others.

Reply:
The arguments will be passed to the atomic function, what's a possible incorrect usage? I feel like even if it's malformed it'll be contained to the function argument scope.